k8s/namespaces/neuvector/patches/ithc/cluster-00/neuvector.yaml

```diff
@@ -67,3 +67,16 @@ spec:
             traefik.ingress.kubernetes.io/router.tls: "true"
           host: sds-neuvector00.ithc.platform.hmcts.net
           enabled: true
+      cve:
+        affinity:
+          podAntiAffinity:
```

Comment:
this is pod anti affinity, we want node anti affinity and it should be against the system node pool, see:
https://github.com/hmcts/aks-sds-deploy/blob/b1caab9a0777e4bad183414bee3a5b21c308eac3/kubernetes/charts/fluxcd/flux-values.yaml

also once it's working it should be moved to our helm chart (chart-neuvector) where we set some defaults on top of the upstream chart